src/evernode-common.js

```diff
@@ -68,9 +68,8 @@ const HookStateKeys = {
     MAX_TOLERABLE_DOWNTIME: "455652010000000000000000000000000000000000000000000000000000000A",
     MOMENT_TRANSIT_INFO: "455652010000000000000000000000000000000000000000000000000000000B",
     MAX_TRX_EMISSION_FEE: "455652010000000000000000000000000000000000000000000000000000000C",
-    REGISTRY_ADDR: "455652010000000000000000000000000000000000000000000000000000000D",
-    HEARTBEAT_HOOK_ADDR: "455652010000000000000000000000000000000000000000000000000000000E",
-
+    REGISTRY_ADDR: "455652010000000000000000000000000000000000000000000000000000000E",
```

Comment:
Shall we change the order of new state keys inside  `HookStateKeys` ?  `45565201....D` followed by `45565201...E`